### DISCUSSION TITLE
Localisation\cs

### DISCUSSION THREAD

**Discussion by mosnerrene** [supermerill/SuperSlicer#2691] [Q&A]
Podle vašeho návodu jsem přeložit SuperSlicer na cs czech neni to dokonalé ale pro mě dostačující. 
Chcete někam uložit vytvořené soubory?
A chci se také zeptat jestli mohu poskytnout vytvořený překlad  SuperSlicer.mo přátelům?
Nerad bych porušil autorská práva nebo licence. Dekuji.

**Comment by renemosner** [comment]
According to your instructions, I translate SuperSlicer to cs czech is not perfect but sufficient for me.
Do you want to save the created files somewhere?
And I also want to ask if I can provide the created translation of SuperSlicer.mo to friends?
I would not like to revoke a copyright or license. Thank you.

**Reply by supermerill** [reply]
no problem.
If you want, i can integrate your changes. Just send me the .po

**Reply by renemosner** [reply]
> žádný problém. Pokud chcete, mohu integrovat vaše změny. Stačí mi poslat .po

https://github.com/renemosner/Translation-of-SuperSlicer.git

Snad je to takto v pořádku. 
A mohu mnou zkompilované MO poskytnout přátelům?

**Reply by renemosner** [reply]
Did you look at the translated cs?
And did it help or is it unusable?

**Reply by supermerill** [reply]
sorry, didn't had the time.
I integrated your additions into the next nightly / release
